Bill3.java

```diff
@@ -1,8 +1,11 @@
-// Splits a restaurant bill evenly among three diners.
 public class Bill3 {
 	public static void main(String[] args) {
-		// To get you started, here is the first line in the program:
 	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+		String name2 = args[1];
+		String name3 = args[2];
+		int bill = Integer.parseInt(args[3]);
+		double SplitsBill = bill/3;
```

Comment:
1. Names should be in camelCase format.
2. Spacing
3. int / int = int -> 5 / 2 => 2 => 2.0 
4. Per the request of the assignment (see PDF) 
the value needed was the ceil value (rounded up)

double splitsBill = Math.ceil(bill / 3.0);